pom.xml

```diff
@@ -264,6 +269,10 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-deploy-plugin</artifactId>
                     <version>2.8.2</version>
+                    <configuration>
+                        <uniqueVersion>false</uniqueVersion>
```

Comment:
What does this do? [Plugin page](https://maven.apache.org/plugins/maven-deploy-plugin/deploy-mojo.html) doesn't let you see doc for older versions, and most recent version doesn't have the property.

Reply:
https://maven.apache.org/plugins/maven-deploy-plugin/deploy-file-mojo.html#uniqueVersion  . I put that in case uploading the same version name causes the name to somehow change